python/mxnet/recordio.py

```diff
@@ -175,6 +177,10 @@ def pack_img(header, img, quality=80, img_fmt='.JPEG'):
         The packed string
     """
     assert opencv_available
-    ret, buf = cv2.imencode(img_fmt, img, [cv2.IMWRITE_JPEG_QUALITY, quality])
+    if img_fmt is '.jpg':
```

Comment:
also use == other than is
Now the codes will never be executed. 
Please test your code before PR especially the core parts...